spec/chapters/controllers.tex

```diff
@@ -329,3 +329,17 @@ \section{Views}
 depending on which controller from Section \ref{models} triggered this view's 
 processing.
 
+\subsection{Building URIs in a View}
+\label{mvc_uri}
+
+In {\tt views} links and form actions require a URI. To avoid repeating the declarative
+mapping to URIs on controller methods MVC provides a way to build URIs from the
+{\tt MvcContext}:
+
+\begin{listing}{1}
+${mvc.uri('MyController#myMethod' {'id': 42, 'foo': 'bar'})}
+\end{listing}
+
+The controller method can either be identified by the simple name of the controller class
```

Comment:
As this section is rather short, we should add something like:

"Please refer to the javadocs of `MvcContext` for a full description of the different ways to provides parameter values for building URIs."

Or something similar. Thoughts?